### ISSUE TITLE
Uncaught Error: closure invoked recursively or destroyed alread

### ISSUE THREAD

**Issue by dmvict** [Nazariglez/notan#119]
I tried to make a little web app using the example `assets_load_texture` and got the error `Uncaught Error: closure invoked recursively or destroyed already`. 

I've built the examples and run the example `assets_load_texture` and got the same error. 

Maybe, the problem is similar to that solved in https://github.com/ryanisaacg/platter/pull/5. 

Please, help me to solve the bug.

**Comment by Nazariglez** [comment]
Hey hello! 

It may be related to the issue that you shared but it kind of weird because I don't leak (I store them) the callbacks on wasm.

Let's test this, does this example work for you? https://nazariglez.github.io/notan-web/examples/assets_load_texture.html 

What browser and SO are you using?

**Comment by Nazariglez** [comment]
I can reproduce it, it seems weird because I am even testing old versions that were working, and right now it doesn't. Thanks for the heads up, I'll fix it asap. Meanwhile, you can use `include_bytes` to load the texture as a workaround.

**Comment by Nazariglez** [comment]
I can't reproduce it, I thought that I did but I was loading an asset that retrieves a 404 error. Could this be your case? I see an error on the console related to a callback but it doesn't affect the execution of the app. Can you give me more info? Anything, is your app crashing? SO? Browser? notan version used? Anything that can help me to test this. 

Thanks

**Comment by dmvict** [comment]
The error does not crash the app and can be reproduced only on example `assets_load_textute`. 
Please, open the example https://nazariglez.github.io/notan-web/examples/assets_load_texture.html, open the browser console and you should get such output
![image](https://user-images.githubusercontent.com/47529590/183008283-2097719e-9a2f-4596-b93b-58ffa2f5bfb2.png)

Locally, I've patched your dependency `platter` in `./crates/notan_app` with 
```toml
platter = { git = "https://github.com/TannerRogalsky/platter.git", branch = "leak_closure", features = ["web-sys"] }
```
And the error disappeared. 
Please, update the dependency after the PR https://github.com/ryanisaacg/platter/pull/5 will be merged, and the new version will be released.

**Comment by Nazariglez** [comment]
I see what you mean now.

I don't think that this PR will be merged soon, it was open in 2020 and is still there. The best option we have (IMO) is to fork platter, and create a new crate called platter2 with the fix. I'll take care of it this weekend.

**Comment by Nazariglez** [comment]
Done, it will be published with v0.6.0, meanwhile, you can keep using 0.5.1 due that it's not a critical error (only a log message), or use the develop branch from Github, just FYI, this branch can break at any time.

Thanks for discovering this.

**Comment by dmvict** [comment]
> Done, it will be published with v0.6.0, meanwhile, you can keep using 0.5.1 due that it's not a critical error (only a log message), or use the develop branch from Github, just FYI, this branch can break at any time.
> 
> Thanks for discovering this.

Thank you for your work.
